### ISSUE TITLE
Fullscreen doesnt work with local songs

### ISSUE THREAD

**Issue by PJGitHub9** [huhridge/huh-spicetify-extensions#4]
For some reason full screen doesnt work with local songs. It just shows up with a grey frosted box in the middle of the screen and only goes away if i click 'escape' on my keyboard.

**Comment by huhridge** [comment]
try the latest commit, lemme know if it doesn't work

**Comment by PJGitHub9** [comment]
Still doesnt work, where the album art should be its frosted and the buttons appear. but the background doesnt change or anything

**Comment by huhridge** [comment]
can you send me ss of console?

**Comment by PJGitHub9** [comment]
Yeah, so i downloaded the zip file from the repo. Then copied the .js file to '\.spicetify\Extensions' and ran 'spicetify apply'. It still doesnt work. 

Im fairly new to spicetify. how do i run the console?

**Comment by huhridge** [comment]
run `spicetify enable-devtool` and then press Ctrl+Shift+i and then select Console in top bar

**Comment by PJGitHub9** [comment]
![Capture](https://user-images.githubusercontent.com/45070803/151987193-0854aef6-e61d-49dc-a6ab-6c901657767c.PNG)
This is the console screenshot

**Comment by huhridge** [comment]
open the full app display with local file and then send me the ss

**Comment by PJGitHub9** [comment]
![Capture](https://user-images.githubusercontent.com/45070803/151989796-36492c26-9624-4cad-a66a-05d7697119d1.PNG)

**Comment by huhridge** [comment]
can you go to sources and open fullappdisplaymod.js, it should be here and scroll to line 40 and show me
![Screenshot 2022-02-01 at 20 17 55](https://user-images.githubusercontent.com/67046436/151990498-a2ff08de-f5c1-4931-87c7-15cdcb873931.jpg)
r

**Comment by PJGitHub9** [comment]
![image](https://user-images.githubusercontent.com/45070803/151990944-6a292f14-560a-4552-9ab7-4f1eb8186eb1.png)

**Comment by huhridge** [comment]
Can you try the latest commit?

**Comment by PJGitHub9** [comment]
Awesome!! It worked!! :D  Thank you so much! Love the fullscreen extension, its my go to 😎
